scripts/qemu/ia32-generic-ata.sh

```diff
@@ -8,8 +8,8 @@
 
 # Add ATA drives
 while [ "$#" -gt 0 ]; do
```

Comment:
For varable `$#` double quote is not needed ( it is a number )

Reply:
Added in next commit.